Modules/SubModule_Import/xASL_imp_NII2BIDS_Subject.m

```diff
@@ -77,11 +75,17 @@
     % Iterate over aliases
     if ~isempty(visitAliases)
         for iAlias = 1:numel(visitAliases)
-            checkExpression = regexp(nameSubjectSession, [separator visitAliases{iAlias,1} '$'], 'once');
-            if ~isempty(checkExpression) % nameSubject should end in the visit alias
-                visitName = nameSubjectSession(checkExpression:end);
-                subjectName = nameSubjectSession(1:checkExpression-1);
-            end
+			if ~isempty(visitAliases{iAlias,1})
+				if visitAliases{iAlias,1}(1) == separator
+					checkExpression = regexp(nameSubjectSession, [visitAliases{iAlias,1} '$'], 'once');
```

Comment:
Do we consistently have visit/session in legacy and session/run in BIDS?

Reply:
No. we don't have this consistently. It was a mess and we tried to fix it a bit. but right now, it's a mess both in variable names and comments :) Please make an issue for these two things ;)

Reply:
Please don't, we already have issues AFAIK 😆